src/app/admin/report/[term]/[name]/_components/Report.tsx

```
@@ -0,0 +1,106 @@
+  return (
+    <Flex
+      alignItems="center"
+      className="w-fit rounded-xl border-[1.5px] border-gray-100 bg-gray-100 p-1 pr-2.5"
+    >
+      <Icon name="locate" size={25} />
+      {place?.trim() ? <Body3>{place}</Body3> : <EmptyText text="장소 없음" />}
+    </Flex>
+  );
+}
+
+function Date({ start, end }: { start?: string; end?: string }) {
```

Biome (2.1.2)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)